Index: src/App/Connect.java
===================================================================
diff --git a/src/App/Connect.java b/src/App/Connect.java
--- a/src/App/Connect.java	
+++ b/src/App/Connect.java	
@@ -16,7 +16,7 @@
             // Java Driver for connecting MySQL Database
             Class.forName("com.mysql.cj.jdbc.Driver").newInstance();
             // port number is 3306
-            connect = DriverManager.getConnection("jdbc:mysql://localhost:8889/epmtdb","root","root");
+            connect = DriverManager.getConnection("jdbc:mysql://localhost/epmtdb","root","");
 
         } catch (Exception e) {
             e.printStackTrace();
